packages/integration-karma/test/template/properties/attrs/uppercaseChild/uppercaseChild.html

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
This test ensures that a parent component can pass an attribute, with a leading hyphen, to a child component via template and that its child component correctly receives the value (note that child's public property starts with an uppercase character). I don't believe this can be verified in the fixture.

Reply:
The generated template code should be sufficient: https://github.com/salesforce/lwc/pull/1687/files#diff-9be0c45d6c1984726cb1381b11a723d8R12-R18. I don't have a strong opinion, so if you feel that you want to keep it, that's fine.